packages/sources/vue/src/components/overlays/VtmnToast/VtmnToast.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import '@vtmn/css-toast/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnToast',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    withIcon: {
+      type: Boolean,
+      default: false,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+    withCloseButton: {
+      type: Boolean,
+      default: false,
+    },
+    timeout: {
+      type: Number,
+      default: 8000,
+    },
+    closeCallBack: {
+      type: Function as PropType<Function>,
+      required: true,
+    },
+  },
+  setup(props) {
+    props = reactive(props);
+
+    const handleClose = () => {
+      return props.closeCallBack;
+    };
+
+    return {
+      classes: computed(() => ({
+        'vtmn-toast': true,
+        show: true,
+        [`vtmn-toast--with-icon-info`]: props.withIcon,
+      })),
+      handleClose,
+    };
+  },
+});
+</script>
+
+<template>
+  <div :class="classes" role="status" v-bind="$attrs">
+    <div class="vtmn-toast_content">{{ content }}</div>
+    <VtmnButton
+      v-if="withCloseButton"
+      iconAlone="close-line"
+      variant="ghost-reversed"
+      size="small"
+      aria-label="close"
+      @click.prevent="handleClose"
```

Comment:
Same as the VtmnSearch component, I think it's better to emit a 'close' event instead of using a callback function.

You can remove the closeCallBack props and add the 'close' event to emits declaration.